app/models/setting/content.rb

```diff
@@ -137,6 +137,9 @@ def self.load_defaults
                  N_("If hosts fail to register because of duplicate DMI UUIDs " \
                     "add their comma-separated values here. Subsequent registrations will generate a unique DMI UUID for the affected hosts."),
                  [], N_('Host Duplicate DMI UUIDs')),
+        self.set('host_profile_steal', N_("Allow new Host registrations to overtake registered profiles with matching hostname " \
```

Comment:
how about "host_profile_assume"  and: "Allow new Host registrations to assume registered profiles with matching hostname as long as the registering DMI UUID is unique."

Reply:
updated!